test/e2e/tests/settings/state-logs.json

```diff
@@ -69,6 +69,7 @@
     "onboardedInThisUISession": "boolean",
     "openMetaMaskTabs": {},
     "pendingTokens": {},
+    "pna25Acknowledged": "boolean",
```

Comment:
mmm interesting. It seems this key should also go inside the metamask object (a few lines below). This will fix the current test failure for firefox (the state-logs test is only run on firefox). I'm unsure if that's expected though, to have the key duplicated in 2 places in the state logs: `metamask` and `appState` 🤔 
See test logs for reference:

<img width="1226" height="204" alt="Image" src="https://github.com/user-attachments/assets/7bc272ac-9a89-46f9-bac6-5d97623340e4" />

I've tried locally and indeed the key appears both places.